marpa/Cargo.toml

```diff
@@ -8,7 +8,10 @@ repository = "https://github.com/jrobsonchase/marpa"
 documentation = "https://docs.rs/marpa"
 edition = "2018"
 
+[lib]
+crate-type = ["lib"]
+
 [dependencies]
 regex = "1.1.0"
 regex-syntax = "0.6.4"
-libmarpa-sys = "0.1.3"
+libmarpa-sys = {path="../libmarpa-sys"}
```

Comment:
Whoops, could've sworn I converted this to a path dep! Maybe it got accidentally reverted or something.

Anyway, we should actually use *both*, that way it uses the path when developing locally, but can still be pushed to crates.io:

```toml
[dependencies.libmarpa-sys]
version = "0.1.3"
path = "../libmarpa-sys"
```